workflow/rocoto/gfs_tasks.py

```diff
@@ -902,7 +902,7 @@ def awips_20km_1p0deg(self):
     def awips_g2(self):
 
         deps = []
-        dep_dict = {'type': 'metatask', 'name': f'{self.cdump}atmprod'}
+        dep_dict = {'type': 'metatask', 'name': f'{self.cdump}post'}
```

Comment:
Something that needs to be updated from the merge commit
```suggestion
        dep_dict = {'type': 'metatask', 'name': f'{self.cdump}atmprod'}
```